vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -68,6 +65,9 @@ type Serializer struct {
 var _ runtime.Serializer = &Serializer{}
 var _ recognizer.RecognizingDecoder = &Serializer{}
 
+// TODO: Remove jsoniter for now as it is buggy and fails the serialization
+// test for DockerImage objects.
+/*
```

Comment:
Can you add more info to the commit message? What fails? Do we have a test-case? Can we open an issue in the upstream project?

Reply:
@mfojtik fyi, since you've touched it